recipes/mastercard/runner.sh

```diff
@@ -6,26 +6,39 @@ NAME=$(basename $BASEDIR)
 VERSION=$DATE
 
 
+AWS_DEFAULT_REGION=us-east-1
+
+
+
 (      
     cd $BASEDIR
     mkdir -p input
     mkdir -p output
     #was having trouble writing to input. Input directory is temporary and will not persist.
     chmod 777 input
     
-    #check to verify there is only one file on the mastercard server. 
-    #Files will not download and delete unless there is only one zip file. 
+    #check to verify there is a file on the mastercard server.
+    #Files will not download and delete unless there is at least one zip file. 
+    #decision to not check for single file.
     #comment out for texting
     ROWCOUNT=$(echo 'ls -l' | sftp -q -oPort=22022 -o StrictHostKeyChecking=no -i ~/.ssh/id_rsa_axway newyorkcity@files.mastercard.com:geoinsights/data/fromMC | grep .zip | wc -l)
```

Comment:
what if there are multiple zip files? which row count is it going to capture?

Reply:
the row count is the number of lines returned with the ls command. If there is one file, ROWCOUNT=1. Two files, ROWCOUNT=2.